### ISSUE TITLE
CVE-2018-8292

### ISSUE THREAD

**Issue by antoniore-edw** [petabridge/petabridge.cmd-issues#50]
The package uses a vulnerable version of System.Net.Http which is affected by [CVE-2018-8292](https://nvd.nist.gov/vuln/detail/cve-2018-8292).

**Comment by Aaronontheweb** [comment]
I don't think this _really_ affects us as we never make any HTTP calls at all inside Petabridge.Cmd (it's all Akka.IO-based, which uses TCP sockets) but we'll see if we can make the warning go away, although I don't even know what dependency we take that could cause this.

**Comment by Arkatufus** [comment]
System.Net.Http was a dependency of:
* v4.1.0:
  * Akka.Streams -> Reactive.Streams
* v4.3.0:
  * FsCheck.Xunit
  * xunit
  * Akka.Cluster -> Akka.Remote -> DotNetty.Handlers
  * Phobos.Actor.Common
  * Akka.Remote -> DotNetty.Handlers
  * Akka.Cluster.Sharding -> Akka.Cluster -> Akka.Remote -> DotNetty.Handlers
  * Akka.Cluster.Hosting -> Akka.Cluster -> Akka.Remote -> DotNetty.Handlers

Patched System.Net.Http version: 4.3.4
